README.md

```diff
@@ -117,33 +126,25 @@ Subscribing to DOM events is similar:
 React.findDOMNode(this).addEventListener('change', function(e){...});
 ```
 
-## Using `attachedCallback` (Added in 0.8.0)
-
-By default this module uses the web component `createdCallback`, so it kicks off
-the rendering process _before_ the component is fully attached to the DOM. In
-some cases you might prefer to delay the rendering process until the component
-is attached.
-
-If this is the case, you can pass `renderOnAttached` as an option:
-
-```js
-document.registerReact('example-component', 'ExampleComponent', {
-  renderOnAttached: true,
-});
-```
-
 ## Dependencies
 
 * [React.js](https://github.com/facebook/react)
-* `function.bind` and `Object.create` support or polyfills
+* [React DOM](https://github.com/facebook/react)
 * Custom elements support or
-  [polyfill](https://github.com/WebReflection/document-register-element)
-
-## Custom elements polyfill behavior
-
-Polyfill binds to `DOMContentLoaded` in order to process DOM so no custom
-elements exist until it fired. To prevent this hook into `DOMContentLoaded` and
-operate elements on it or after.
+  [polyfill](https://github.com/WebComponents/webcomponentsjs)
+* Support or [polyfills](https://github.com/zloirock/core-js) for:
+  * `regexp.match`
+  * `regexp.replace`
+  * `object.define-setter`
+  * `object.define-getter`
+  * `object.define-property`
+  * `function.name`
+  * `web.dom.iterable`
+  * `array.iterator`
+  * `object.keys`
+  * `object.set-prototype-of`
+  * `reflect.construct`
+  * `function.bind`
```

Comment:
It'd be nice to maybe include a demo of a setup that works in IE11.

I wonder if we could write some Karma tests and run them on BrowserStack in IE11/Chrome/FF/Safari to help test cross browser correctly.

Reply:
https://github.com/uPortal-contrib/uPortal-web-components/blob/a5f2c2d7208c9046eb7e43b4a3be61a9fbdd1bfb/docs/en/components/waffle-menu/demo.html#L60-L64 is the demo I currently have, something like this could be added into the documentation here.

Reply:
> I wonder if we could write some Karma tests and run them on BrowserStack in IE11/Chrome/FF/Safari to help test cross browser correctly.

Probably, it sounds like an effort significant enough that it deserves its own ticket and PR.

Reply:
For sure - don't think this is worth doing in this PR!